### PR TITLE
🌳 CRT progress widget

### PR DESCRIPTION
**Pull request by WRadoslaw**
#3389

### CODE REVIEW

File: packages/atlas/src/components/ProgressWidget/ProgressWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+export const ProgressWidget = ({ steps, activeStep, goalComponent }: ProgressWidgetProps) => {
+  const [isVisible, setIsVisible] = useState(false)
+  const [glider, setGlider] = useState<SwiperInstance | null>(null)
+  const drawer = useRef<HTMLDivElement>(null)
+  const xsMatch = useMediaMatch('xs')
+  const smMatch = useMediaMatch('sm')
+  const isDone = activeStep + 1 > steps.length
+  return (
+    <div style={{ position: 'relative' }}>
```
Comment (attemka) [inline packages/atlas/src/components/ProgressWidget/ProgressWidget.tsx]: would look better with a styled

File: packages/atlas/src/components/ProgressWidget/ProgressWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+      <Header progressWidth={isVisible ? '0%' : `${Math.round((activeStep / steps.length) * 100)}%`}>
```
Comment (attemka) [inline packages/atlas/src/components/ProgressWidget/ProgressWidget.tsx]: you have a `getProgressPercentage` func

File: packages/atlas/src/components/ProgressWidget/ProgressWidget.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+const Template: StoryFn<ProgressWidgetProps> = (args) => <ProgressWidget {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  activeStep: 2,
+  steps,
+  goalComponent: (
+    <Text variant="t200" as="p">
+      Complete 1 more step to achive
```
Comment (attemka) [inline packages/atlas/src/components/ProgressWidget/ProgressWidget.stories.tsx]:
```suggestion
      Complete 1 more step to achieve{' '}
```

File: packages/atlas/src/components/ProgressWidget/ProgressWidget.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+export default {
+  title: 'CRT/ProgressWidget',
+  component: ProgressWidget,
+} as Meta<ProgressWidgetProps>
+
+const steps: ProgressWidgetProps['steps'] = [
+  {
+    title: 'Create token',
+    description: 'Createn own token and share it with your viewers!',
```
Comment (attemka) [inline packages/atlas/src/components/ProgressWidget/ProgressWidget.stories.tsx]:
```suggestion
    description: 'Create own token and share it with your viewers!',
```

File: packages/atlas/src/components/ProgressWidget/ProgressWidget.styles.ts [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  gap: ${sizes(10)};
+  padding: ${sizes(4)};
+`
+
+export const ProgressBar = styled.div<{ progress: number }>`
+  position: relative;
+  height: 12px;
```
Comment (attemka) [inline packages/atlas/src/components/ProgressWidget/ProgressWidget.styles.ts:68]: I think I've asked before (maybe I've asked Bartosz), but why won't use sizes here and in other hardcoded dimensions?

**Review by attemka — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/ProgressWidget/ProgressWidget.styles.ts (hunk 5 above)
Comment (WRadoslaw) [reply packages/atlas/src/components/ProgressWidget/ProgressWidget.styles.ts:68]: The way it was described to me is that we should use `sizes` for spacing purposes, eg. gaps, paddings, and margins.

**Review by attemka — APPROVED** [review]

**Pull request merged**